### ISSUE TITLE
Full End-to-End Smoke Test from Training to Evaluation on SQuAD

### ISSUE THREAD

**Issue by Malikeh97** [Malikeh97/CSC2552-Final-Project#6]
- [ ] Started from pretrained enc and dec [BASELINE] 
- [ ] Check out the eval loss
- [ ] Eval on FineInstruct benchmarks + SQuAD Validation Set
